.github/workflows/sonarcloud-scan.yml

```diff
@@ -88,3 +88,17 @@ jobs:
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
         with:
           projectBaseDir: packages/linters/
+
+      - name: Check coverage directory - http-front-cache
+        run: |
+          chmod +x ./.github/workflows/check-coverage.sh
+          ./.github/workflows/check-coverage.sh http-front-cache
+
+      - name: SonarCloud Scan - http-front-cache 🚨
```

Comment:
I have already added this, but the package will be added in the next PR.

Reply:
the PR to create the lib: https://github.com/juntossomosmais/time-out-market/pull/267